src/makeCalculator.js

```diff
@@ -4,7 +4,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
+
+  return {
+    add(a, b) {
+      return a + b;
+    },
+    subtract(a, b) {
+      return a - b;
+    },
+    multiply(a, b) {
+      return a * b;
+    },
+    divide(a, b) {
+      if (b === 0) {
+        return 'Cannot divide by zero';
+      }
+
+      return a / b;
+    },
+    operate(operation, number) {
+      result = operation(result, number);
+
+      return this;
+    },
+    reset() {
+      result = 0;
+
+      return this;
+    },
+    get result() {
+      return result;
```

Comment:
[NAMING] - The result property should not be a getter method. According to the checklist, the result should be a directly accessible property.